server/utils.js

```diff
@@ -78,3 +79,17 @@ utils.decodeSpecialChar = function(url) {
   url = url.replace(/%3F/g, '?');
   return url;
 };
+
+/**
+ * Prints logs with timestamps.
+ * @param {!Array<string>} args Content to be logged.
+ */
+utils.serverLog = function(args) {
+  if (!args.length) {
+    return;
+  }
+  var date = new Date();
```

Comment:
It will be better to write both the posix time (int) and the server local time (date string).